### PR TITLE
Show different column set for Simbad results & rearrange column order

### PR DESCRIPTION
**Pull request by rpiaggio**
Also: simplify sticky cols styles by using previous sibling selector (no more `with-`).

### CODE REVIEW

File: common/src/main/webapp/less/style.less [hunk 1]
```diff
@@ -527,6 +527,19 @@ html {
 
 .explore-target-summary-delete {
   left: 0;
+  overflow: hidden;
```
Comment (cquiroz) [inline common/src/main/webapp/less/style.less:530]: Should we move these to scss?

**Review by cquiroz — CHANGES_REQUESTED** [review]
Looks fine to me but I'd like that when we need to change the less file we rather convert it to scss

**Comment by rpiaggio** [comment]
Good idea, I'll add another commit with that

**Comment by cquiroz** [comment]
Do you have a screenshot?

**Comment by rpiaggio** [comment]
> Do you have a screenshot?

image

image

Simbad doesn't show catalog name/cz/morphology/sed.

**Comment by cquiroz** [comment]
Thanks

**Pull request merged**